contact.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Form</title>
+    <link rel="stylesheet" href="contactStyle.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;600&family=Poppins&family=Roboto&display=swap"
+      rel="stylesheet"
+    /> 
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+</head>
+<body>
+    <section class="contact container">
+        <p class="title"> I'm always  <br>
+            interested in hearing about new projects,so if you'd
+             like to chat please get in touch.</p>
+         <form action="https://formspree.io/f/mdojzzgj" method="post">
+             
+          <input type="text" class="item" name="fullName" placeholder="Full Name" required pattern="[A-Za-z]" maxlength="30">
+            <input type="email" class="item" placeholder="Email address" name="email" required  >
+           
+            <textarea  name="message" id="tetarea" class="item textarea" cols="20" rows="7" placeholder="Enter text here" maxlength="50" required></textarea>
```

Comment:
- [ ] As written in the project requirements. I kindly suggest that you increase your text area maxlength to 500 because the Message field should accepts up to 500 chars